LiskHQ/lisk-sdk
# Update lastBlockId to be bytes and update protocolVersion to networkVersion

### What was the problem?

This PR resolves #5544 and resolves #5528 

### How was it solved?

- Remove application state as it was just proxying the information between node and network
- Update lastBlockId => lastBlockID and change the type to `bytes`
- Update all remaining protocolVersion to networkVersion

### How was it tested?

- Update all the test
- Connect 2 nodes and see if they have valid information
